core/txpool/locals/tx_tracker.go

```diff
@@ -88,6 +88,11 @@ func (tracker *TxTracker) TrackAll(txs []*types.Transaction) {
 		if tx.Type() == types.BlobTxType {
 			continue
 		}
+		// Ignore the transactions which are failed for fundamental
+		// validation such as invalid parameters.
+		if tracker.pool.ValidateTxBasics(tx) != nil {
```

Comment:
We should maybe add tracing log here, so we can see why a transaction was not tracked

Reply:
What kind of log do you prefer? Info or Debug?

I am a bit concerned the Info logs might be overwhelming for certain types of node, e.g. rpc node.

Reply:
Debug